sopel/privileges.py

```diff
@@ -0,0 +1,123 @@
+"""Constant for user privileges in channel.
+
+Privilege levels
+================
+
+Historically, there were two user privileges in channels:
+
+* :data:`OP`: channel operator, or chanop, set and unset by ``+o`` and ``-o``
+* :data:`VOICE`: give the privilege to send message to a channel with the
+  ``+m`` mode, set and uset by ``+v`` and ``-v``
+
+Since then, other privileges have been used by IRC servers and clients:
+
+* :data:`HALFOP`: intermediate level between Voiced and OP, set and unset by
+  ``+h`` and ``-h``
+* :data:`ADMIN`: channel admin, above OP and below OWNER, set and unset by
+  ``+a`` and ``-a``
+* :data:`OWNER`: channel owner, above ADMIN and OP, set and unset by ``+q`` and
+  ``-q``
+
+.. important::
+
+    Not all IRC networks support these added privilege modes. If you are
+    writing a plugin for public distribution, ensure your code behaves sensibly
+    if only +v (voice) and +o (op) modes exist.
+
+Compare privileges
+==================
+
+Privileges are powers of two, the bigger the higher the privileges::
```

Comment:
```suggestion
This module represents privileges as powers of two, with higher values assigned
to higher-level privileges::
```

It was this or a semicolon, and this reads cleaner to me.